test/dpul_collections_web/live/search_live_test.exs

```diff
@@ -53,27 +53,38 @@ defmodule DpulCollectionsWeb.SearchLiveTest do
   end
 
   test "GET /search renders thumbnails for each resource", %{conn: conn} do
-    {:ok, view, _html} = live(conn, "/search?")
+    {:ok, _view, html} = live(conn, "/search?")
 
-    assert view
-           |> has_element?(
-             "#item-1 img[src='https://example.com/iiif/2/image1/square/350,350/0/default.jpg']"
-           )
+    {:ok, document} =
+      html
+      |> Floki.parse_document()
 
-    assert view
-           |> has_element?(
-             "#item-1 img[src='https://example.com/iiif/2/image2/square/350,350/0/default.jpg']"
-           )
+    # There should be a maximum of 5 thumbnails on the search results page
+    assert document |> Floki.find("#item-1 > div > img") |> Enum.count() == 5
 
-    assert view
-           |> has_element?(
-             "#item-2 img[src='https://example.com/iiif/2/image1/square/350,350/0/default.jpg']"
-           )
+    # Odd numbered documents do not have a thumbnail id so the order of
```

Comment:
Can you clarify that this comment is about the test data as opposed to the implementation?

Reply:
Done